Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -17,15 +18,10 @@ class PetSingleDetailsViewController: UIViewController {
     @IBOutlet weak private var saveSinglePetButton: UIButton!
 
     // MARK: - Vars/Lets
-
+    private let viewContext = (UIApplication.shared.delegate as? AppDelegate)?.persistentContainer.viewContext
+    private lazy var singlePetViewModel = SinglePetViewModel()
     private var singlePet: AdoptPet?
```

Comment:
`ViewController` should not really know about this either.

Reply:
I will remove this logic 👍🏻